app/api/group/create/route.ts

```diff
@@ -0,0 +1,18 @@
+import {createGroup} from '@/service/pads/GroupManager';
+import { APIResponse } from '@/types/APIResponse';
+import { NextResponse } from "next/server";
+
+export async function POST() {
+  const groupId = await createGroup();
+
+  const response = {
+    code: 0,
+    data: {
+      groupId
+    },
+    message: 'ok'
+  } satisfies APIResponse;
+
+  return NextResponse.json(response);
```

Comment:
```suggestion
   return Response.json(response, { status: 200 });
```

[mdn ref](https://developer.mozilla.org/en-US/docs/Web/API/Response)